src/en/get-started/get-started.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Get started
+translationKey: getstarted
+layout: 'layouts/base.njk'
+eleventyNavigation:
+  key: getstartedEN
+  title: Get started
+  locale: en
+  order: 1
```

Comment:
The mock up has this third after page templates, just checking if this is the proper position for the page in the top nav